### ISSUE TITLE
Kubeconfig file modified when using Azure OIDC

### ISSUE THREAD

**Issue by juhtie01** [robscott/kube-capacity#24]
We have started to use Azure OIDC to authenticate to our clusters. Whenever I execute `kube-capacity` command for a cluster, it modifies my `kubeconfig` file and removes a setting (`environment: AzurePublicCloud`) which makes `kubectl` not work with the authentication anymore.

Original content of `kubeconfig`:
```yaml
...
users:
- name: oidc_user
  user:
    auth-provider:
      config:
        access-token: <TOKEN>
        apiserver-id: <APISERVER ID>
        client-id: <CLIENT ID>
        environment: AzurePublicCloud
        expires-in: "3599"
        expires-on: "1579869933"
        refresh-token: <REFRESH TOKEN>
        tenant-id: <TENANT ID>
      name: azure
...
```

`kubeconfig` contents after running `kube-capacity`:
```yaml
...
users:
- name: oidc_user
  user:
    auth-provider:
      config:
        access-token: <TOKEN>
        apiserver-id: <APISERVER ID>
        client-id: <CLIENT ID>
        expires-in: "3599"
        expires-on: "1579869933"
        refresh-token: <REFRESH TOKEN>
        tenant-id: <TENANT ID>
      name: azure
...
```

Same bug seems to be in Stern: https://github.com/wercker/stern/issues/119

A [Stern fork](https://github.com/evenh/stern#fork-status) seems to have fixed the issue by upgrading to a newer Kubernetes API.

**Comment by robscott** [comment]
Hey @juhtie01, thanks for the heads up! I'll try to find some time to update the dependencies here to fix this.

**Comment by robscott** [comment]
Hey @juhtie01, this should be fixed in 0.4.0, let me know if you run into issues. Thanks!

**Comment by juhtie01** [comment]
Works perfectly. Thank you!
